client/src/components/NavBar/index.tsx

```diff
@@ -7,11 +7,14 @@ import { EVENTS } from "../../analytics/events";
 import { ROUTES } from "./routes";
 import Icon from "../icon/icon";
 import { Left, LinkWrapper, MainWrapper, Nav, Right, Wrapper } from "./style";
+import { NavDivider } from "./NavDivider/style";
```

Comment:
i'm pretty sure this is wrong - for some reason i'm not able to add a global import like `src/components/NavBar/components/NavDivider`, and this is the only import i was able to actually add. but it is almost certainly wrong to import from the styled page right?

Reply:
Ah I see your `NavDivider/index.tsx` doesn't have an export, so that's probably why you can't import it from `src/components/NavBar/components/NavDivider`. 

On the Data Portal side this is what Fran did, so maybe we just need to do the same thing? 
https://github.com/chanzuckerberg/single-cell-data-portal/blob/78783d1685cd14d631caee8da25a58bb780b3343/frontend/src/components/Header/components/Nav/components/NavDivider/index.tsx

Thank you!

Reply:
i think it's a bit more complicated - i [read this blog post](https://dev.to/nilanth/no-more-import-in-react-2mbo) and i believe it has something to do with how `tsconfig.json` in explorer differs from `tsconfig.json` in data portal. specifically, data portal sets `"baseUrl": "."` while explorer doesn't.

i tried playing around with some of the compiler options in explorer, but it ended up being more complicated than i could figure out myself, so i just updated the component declarations here to mimic how explorer does things, which is:
1. to use an arrow function assigned to a constant instead of a named function declaration that we do in data portal
2. uses relative imports rather than absolute imports

Reply:
That sounds good! Yeah personally I prefer function declaration over arrow function, since error stack trace will be more readable. But Explorer does use a lot of arrow functions, so that's fine 😆